samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -669,6 +670,13 @@ void cloud_connection_thread_fn(void)
 		if (connect_cloud()) {
 			LOG_DBG("Monitoring nRF Cloud connection");
 
+			if (IS_ENABLED(CONFIG_NRF_PROVISIONING)) {
+				LOG_INF("Reducing provisioning check interval to %d minutes",
+					CONFIG_POST_PROVISIONING_INTERVAL_M);
+				nrf_provisioning_set_interval(CONFIG_POST_PROVISIONING_INTERVAL_M *
+							      SEC_PER_MIN);
+			}
```

Comment:
I think this should be done inside the `cloud_ready` function rather than directly in the connect loop here